src/Data/Lazy.ss

```diff
@@ -0,0 +1,18 @@
+;; -*- mode: scheme -*-
+
+(library (Data.Lazy foreign)
+  (export defer
+          force
+  )
+
+  (import (only (rnrs base) define lambda)
+  (prefix (purs runtime lib) rt:))
+
+  (define defer
+    (lambda (thunk)
+      (lambda ()
+        (thunk 0))))
```

Comment:
Like I mentioned, I think we could use `'unit` here to at least be somewhat correct.